src/DataTypes/IDataType.h

```diff
@@ -565,4 +565,31 @@ class DataTypeEnum;
 
 template <typename T> inline constexpr bool IsDataTypeEnum<DataTypeEnum<T>> = true;
 
+#define FOR_BASIC_NUMERIC_TYPES(M) \
```

Comment:
Since this is used in `Interpreters/RowRefs.cpp`, can you remove the `#include <AggregateFunctions/Helpers.h>` from there, please? It already includes `IDataType.h` so it should be covered.

Reply:
I've already removed it.